go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/aws/aws-sdk-go-v2 v1.16.2
 	github.com/aws/aws-sdk-go-v2/config v1.15.3
 	github.com/aws/aws-sdk-go-v2/service/ec2 v1.34.0
-	github.com/aws/eks-anywhere-packages v0.1.1
+	github.com/aws/eks-anywhere-packages v0.1.10
```

Comment:
But our release goes to 11. It would be convenient to have this in another PR, but no matter